File: wood-backend/src/models/episode.js
const mongoose = require('mongoose');

const episodeSchema = new mongoose.Schema({
  series: {
    type: mongoose.Schema.Types.ObjectId,
    ref: 'Series',
    required: true,
  },
  episodeNumber: {
    type: Number,
    required: true,
  },
  videoUrl: {
    type: String,
    required: true,
  },
});

module.exports = mongoose.model('Episode', episodeSchema);

File: flutter_wood/lib/models/episode.dart
class Episode {
  final String id;
  final String series;
  final int episodeNumber;
  final String videoUrl;

  Episode({
    required this.id,
    required this.series,
    required this.episodeNumber,
    required this.videoUrl,
  });

  factory Episode.fromJson(Map<String, dynamic> json) {
    return Episode(
      id: json['_id'],
      series: json['series'],
      episodeNumber: json['episodeNumber'],
      videoUrl: json['videoUrl'],
    );
  }
}